frappe/app.py

```diff
@@ -428,6 +429,9 @@ def serve(
 	)
 
 
+# Remove references to pattern that are pre-compiled and loaded to global scopes.
+re.purge()
```

Comment:
`app.py` definitely imports `__init__.py` so is this required?

Reply:
I checked the size of the cache at the end of the init and app modules - There were 100+ and 200+ regexes loaded in each respectively.  I think just running this at the end of app.py would do fine. I expected more regexes to be compiled by init but most modules aren't picked into memory at that point. 

This is one level of a hack. In another, I would prefer to run purge via the ImportLoader. But the better solution would be to never put them in the cache in the first place.